easybuild/easyconfigs/a/Automake/Automake-1.15-fix-brace.patch

```diff
@@ -0,0 +1,10 @@
+--- automake-1.15/bin/automake.in.orig	2015-01-06 03:25:55.000000000 +0800
```

Comment:
@froland Did you create this patch yourself, or did you get it somewhere else?

Please mention the author or upstream reference above this line (anything above this line are comments ignored by `patch`).